packages/opencensus-nodejs/README.md

```diff
@@ -69,6 +69,64 @@ Tracing has many options available to choose from. At `tracing.start()`, you can
 | [`plugin`](https://github.com/census-instrumentation/opencensus-node/blob/master/packages/opencensus-core/src/trace/config/types.ts#L68) | `PluginNames` | A list of trace instrumentations plugins to load |
 | [`exporter`](https://github.com/census-instrumentation/opencensus-node/blob/master/packages/opencensus-core/src/trace/config/types.ts#L70) | `Exporter` | An exporter object |
 
+## Stats
+
+### Collecting stats about your application
+
+OpenCensus Stats allows users to collect measurements an visualize them in many different formats. After instanciated stats, the user needs to define a measure and associate views with it, then, it is possible to record measurements.
+
+```javascript
+// Import and instanciate OpenCensus Stats
+var opencensus = require('@opencensus/core');
+var stats = new opencensus.Stats();
+
+// Defines a measure
+var measure = stats.createMeasureInt64(
+  'my/measure', opencensus.MeasureUnit.UNIT);
+// And associates a view with it
+var tags = {myTagKey: 'myTagValue'};
+var view = stats.createView(
+  'my/view', measure, opencensus.AggregationType.SUM, ['myTagKey'], 'my view');
+
+// Creates a measurement
+var measurement = {measure, tags, value: 10};
+// and records it
+stats.record(measurement);
+```
+
+Similarly for Typescript:
+
+```typescript
+// Import and instanciate OpenCensus Stats
+import {Stats, MeasureUnit, AggregationType} from '@opencensus/core';
+const stats = new Stats();
+
+// Defines a measure
+const measure = stats.createMeasureInt64('my/measure', MeasureUnit.UNIT);
+// And associates a view with it
+const tags = {myTagKey: 'myTagValue'};
+const view = stats.createView(
+  'my/view', measure, AggregationType.SUM, ['myTagKey'], 'my view');
+
+// Creates a measurement
+const measurement = {measure, tags, value: 10};
+// and records it
+stats.record(measurement);
+```
+
+Measures can be of type `Int64` or `DOUBLE`, created by calling `createMeasureInt64` and `createMeasureDouble` respectively. Its units can be:
+
+| MeasureUnit | Usage |
+| ----------- | ----- |
+| UNIT | for general counts |
```

Comment:
I would suggest `codifying` anything in the first column.

Reply:
Great, looks better.